build.gradle

```diff
@@ -12,6 +12,7 @@ repositories {
     jcenter()
     maven { url 'https://repo.jenkins-ci.org/public' }
     mavenLocal()
+    mavenCentral()
```

Comment:
This isn't necessary, `jcenter()` acts as a Maven central cache and provides a number of other artifacts not published to maven central.